omnisharp.el

```diff
@@ -350,7 +350,7 @@ locations in the json."
 cursor at that location"
   (let* ((element-line (cdr (assoc 'Line element)))
          (element-column (cdr (assoc 'Column element)))
-         (element-filename (cdr (assoc 'Filename element)))
```

Comment:
@sp3ctum There's this one instance of Filename with a lower case 'n'.  Was that intentional?